src/translation/translations/en.json

```diff
@@ -58,7 +58,7 @@
     },
     "title": {
       "connectWallet": "Connect a wallet",
-      "yourWallet": "Your wallet"
```

Comment:
@therealemjy let me know if you agree on changing this copy

Reply:
Definitely